src/lib.rs

```diff
@@ -1,18 +1,62 @@
+// Copyright (c) 2020-2021 Swift Navigation Inc.
+// Contact: Swift Navigation <dev@swiftnav.com>
+//
+// This source is subject to the license found in the file 'LICENSE' which must
+// be be distributed together with this source. All other rights reserved.
+//
+// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+// EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+// WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+//! `swiftnav-rs` is a library that implements GNSS utility functions to perform
+//! position estimations. The data used by `swiftnav-rs` typically comes from GNSS
+//! receiver chips as raw observation and ephemeris data. `swiftnav-rs` is more of
+//! a "bring your own algorithm" library, it provides a bunch of functionality that
+//! is useful when processing raw GNSS data, but it provides only limited position
+//! estimation capabilities. Each module encompasses a single set of functionality,
+//! and they are meant to be pretty self-explanatory for developers familiar with
+//! GNSS processing.
+//!
 //! GNSS systems are used to estimate the location of the receiver by determining
-//! the distance between the receiver and several satelites. The satellites send
+//! the distance between the receiver and several satellites. The satellites send
 //! out precisely timed periodic messages and the receiver measures the delay
 //! of those messages. Knowing the location of the satellites at the time of
 //! transmission and the delays of the messages the receiver is able to determine
 //! the location of itself in relation to the satellites.
 //!
-//! Libswiftnav is a library that implements GNSS utility functions to perform
-//! position estimations. The data used by libswiftnav typically comes from GNSS
-//! reciever chips as raw observation and ephemeris data.
+//! `swiftnav-rs` does not provide any functionality for communicating with
+//! receivers made by Swift Navigation, or any manufacturer.
+//! [libsbp](https://github.com/swift-nav/libsbp) is the library to use if you
+//! want to communicate with receivers using Swift Binary Protocol (SBP).
+//!
+//! ## Time
+//! Time is a very important aspect of GNSS. `swiftnav-rs` defaults to representing
+//! all times as GPS times. It provides the ability to manipulate GPS time stamps,
+//! as well as means to convert a GPS time stamp into various other time bases
+//! (GLONASS time, UTC, MJD).
```

Comment:
> as well as means to convert a GPS time stamp into various other time bases
> (GLONASS time, UTC, MJD).

Strictly speaking, this isn't the case yet. I've been slowly working on exposing this conversion functionality. I hope a little lie isn't too much of an issue while I finish the wrapper.